intergov/use_cases/subscribe_by_jurisdiction.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+
+class SubscriptionFailure(Exception):
+    pass
+
+
+class InvalidSubscriptionParameters(Exception):
+    pass
+
+
+class SubscribeByJurisdictionUseCase:
+    def __init__(self, channel_api_url, callback_url, igl_country, secret=''):
+        if not (channel_api_url and callback_url and igl_country):
+            raise InvalidSubscriptionParameters
+        self.channel_api_url = channel_api_url
+        self.callback_url = callback_url
+        self.igl_country = igl_country
+        self.secret = secret
```

Comment:
Secret is expected to be used, otherwise how do we validate pings

Reply:
Where secret should come from?

Reply:
I think it will be a separate S3-backed repository to store these secrets. You generat it (random string) here, subscribe to it, and then message_rx_api will read these secrets and validate websub pings using it. Not sure how they will be stored, may be it's enough to use single secret per channel or rotate them every month or even every re-subscription.

Reply:
Yep, that sounds like a good way to do it @koriaf, but I don't think this is important to get stuff working right now. The alternate way to validate pings is to simply retrieve the message that the ping is about and that is sufficient for the time being. We can improve security once things work as expected.

Reply:
then we can consider this as ready for merge.

Reply:
and BTW current db-channel implementation doesn't cover signing messages anyhow.